ttnn/module.mk

```diff
@@ -35,11 +35,11 @@ endif
 
 ttnn: $(TTNN_LIBS_TO_BUILD)
 
-$(TTNN_LIB): $(TTNN_OBJS) $(TT_DNN_LIB) $(TENSOR_LIB) $(DTX_LIB) $(TT_METAL_LIB) tt_eager/tt_lib
+$(TTNN_LIB): $(TTNN_OBJS) $(TT_DNN_LIB) $(TENSOR_LIB) $(DTX_LIB) $(TT_METAL_LIB) $(TT_LIB_LIB)
```

Comment:
The intent was to keep TTNN_LIB to be a pure NO pybind11 C++ library that completely bypasses any python code.  
Thinking this should be $(TT_LIBS_TO_BUILD) perhaps?

Reply:
I think this change still holds true to your statement, `TT_LIB_LIB` is the c++ ttlib that `tt_eager/tt_lib` resolves to.  It's just that `tt_eager/tt_lib` is a phony target that confuses make into thinking it needs to rebuild TTNN_LIB even when nothing has changed.

Reply:
Hey Eyon, incremental build is a nice QOL, lmk what I can do so move this PR along.